Syntaxes/Markdown (GitHub Italics).tmLanguage

```diff
@@ -3,20 +3,13 @@
 <plist version="1.0">
 <dict>
 	<key>fileTypes</key>
-	<array>
-		<string>md</string>
-		<string>mdown</string>
-		<string>markdown</string>
-		<string>markdn</string>
-	</array>
+	<array/>
 	<key>hideFromUser</key>
 	<true/>
 	<key>injectionSelector</key>
-	<string>(L:text.html.markdown)</string>
-	<key>keyEquivalent</key>
-	<string>^~M</string>
+	<string>(L:text.html.markdown markup.italic.markdown)</string>
 	<key>name</key>
-	<string>Markdown (GitHub Injected)</string>
+	<string>Markdown (GitHub Italics)</string>
```

Comment:
@infininight Still think this name should probably just be `Markdown (GitHub)` as it's doing more than just italics.